### PR TITLE
Update Logo

### PR DESCRIPTION
**Pull request by slandath**
# Update Logo

* Added new logo from LF CS
* Renamed old logo

### CODE REVIEW

**Comment by tgamblin** [comment]
can you remove the .DS_Store files?

**Comment by tgamblin** [comment]
Also needs DCO

**Comment by slandath** [comment]
I'm going to close this and upload the logo using the GitHub website

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 public
+.ds_store
```
Comment (tgamblin) [inline .gitignore:2]: Doesn't this need to be .DS_Store (caps) or does it matter on MacOS?  My global .gitignore has it with caps.

**Pull request closed without merging**